code-challenges.js

```diff
@@ -9,41 +9,84 @@
+//         num1 += numbers[i]
+//         // num one is our place value of where we will start at within the array. they will all add up
+
+//     }
+//     return num1
+//     // this returns the sum of our funtion 
+
+// }
+// console.log(calculateSum(padres1984WorldSeriesRuns))
+// // now were calling our function to add upp all the values within the array of padres1984WorldSeriesRuns
+// console.log(calculateSum(padres1998WorldSeriesRuns))
+// // Explain your code:
+
+// // --------------------2) Create a function that takes in a string and  returns the string with all the characters in reversed order. Use the test variable provided below.
+
+// const lesMiserables = "Jean Valjean"
+// // Expected output: "naejlaV naeJ"
+
+// const lionKing = "Simba"
+// // Expected output: "abmiS"
+// // // Explain your code:
+
+// const reveWords = (words) => {
+//     // we are creating our function name for reversing words within the array 
+//     const splitWords = words.split('') 
+//     // we are creating a function called splitWords that is going into the array and is splitting them to bei individually within the array 
+//     const reversedLetters = splitWords.reverse()
+//     // now that we have our words split individually within the array, we can call on the function .reverse method to reverse the letters within the array
+//     const joinLetters = reversedLetters.join("")
+//     // now that the characters are reversed, we create a function to join them with the .join('') (make sure we have the takcs together or else it will do a different funtions)
+//     return joinLetters
+//     // were returning pour whole function into one with the finished result of all characters split, reversed and joined giving us our desired outcome
+// }
+
+// console.log(reveWords(lesMiserables))
+//     // calling up our words 
+// console.log(reveWords(lionKing))
+
+// // --------------------3) Create a function that takes in an array and returns an array with only the items from the third, fourth, and fifth indexes. Use the test variable provided below.
 
 const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
 // Expected output: [42, 5, -5]
 
 const cohortList = ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
 // Expected output: ["Delta", "Echo", "Foxtrot"]
 
-// Explain your code:
+// // Explain your code:
+const selectItems = function(array) {
+    //were declaring our variable and and calling it selectItems, we use the function array to define a anonymous function that should take the array as an argument and assigns a constant variable selectitems.
+
+    return array.slice(3, 6)
+    // we use the slice method to find only the third through the 5th index of the array of both stock exchange and cohort list?
+}
+
+
+console.log(selectItems(stockExchange))
+// we log it the select items agains the stock exchange to make sure we get teh values within the index we need. 
+console.log(selectItems(cohortList));
```

Comment:
Good work on these Judas!  Can you please comment in your code before you submit, or put all your explanation below each function next time.  It was hard for me to find your code. :)